src/styles/blocks/header.scss

```diff
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: color 0.3s ease;
```

Comment:
Create a variable for the duration.
Also, `ease` is the default function. So you can skip writing it.